tests/e2e/storage_classes.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+
+	"k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/tests/e2e/driver"
+)
+
+func newStorageClass(namespace string, reclaimPolicy v1.PersistentVolumeReclaimPolicy, volumeType, fsType string) *storagev1.StorageClass {
+	fmt.Println("creating storageclass", namespace, volumeType, fsType)
+	provisioner := driver.DriverName
```

Comment:
This is a smell to me.
We should keep the `k8s.io/kubernetes/test/e2e/storage/drivers.DynamicPVTestDriver` interface or maybe create our own with just the methods we need. ie `GetDynamicProvisionStorageClass`.

Reply:
Actually, imo this should be exported by the driver [pkg](https://github.com/mesosphere/aws-ebs-csi-driver/blob/master/pkg/driver/driver.go#L32) because it's used to identify [itself](https://github.com/mesosphere/aws-ebs-csi-driver/blob/master/pkg/driver/identity.go#L29). 

I think I get your point regarding the interface and I'll try another approach.